### ISSUE TITLE
is my 4mm testnet token that on stake is gone after do complete reset? it shouldn't, it should save some where isn't it?

### ISSUE THREAD

**Issue by scraido** [harmony-one/harmony#2570]
**Describe the bug**
Before i'm not elected, and then i can't unstake my staking coin using dashboard or CLI, then i doing "hard reset" removing all, recover back my "ysu" wallet using 12 seed mnemonic passphrase, run the node again, did validator for my another wallet to test another issue, and then i'm back to this "ysu" wallet to unstake it using CLI, now it gave me weird message that tell if i got no token to stake (details on picture)

**To Reproduce**
1. you must have wallet created with new google chrome ext.
2. run the node and tui
3. stake, try to unstake coz you get kickoff from signing
4. stop the node (because you can't unstake using dashboard or CLI, but we focused with CLI here)
5. hard reset (removing all) without succeed unstake using $ sudo rm -rf bootnode .dht* harmony* hmy .hmy* latest lib* md5* node.sh staging *.rlp wallet
6. run the node again
7. restore your wallet using 12 seed mnemonic
4. do unstake using CLI with or without --passphrase

**Expected behavior**
it should you can do unstake even you put "--passphrase" there (just press enter, enter)

**Screenshots**
here the CLI picture update, using enter, enter and without passphrase
![with-without](https://user-images.githubusercontent.com/44559893/77221640-f4c72f80-6b7d-11ea-9758-cfad2ca398e4.png)

but dashboard still show it;
![dashboardshow](https://user-images.githubusercontent.com/44559893/77221707-bbdb8a80-6b7e-11ea-8a74-0498ccc829d7.png)

**Environment (please complete the following information):**
 - OS: ubuntu 18.04
 - Go environment:
GO111MODULE=""
GOARCH="amd64"
GOBIN=""
GOCACHE="/root/.cache/go-build"
GOENV="/root/.config/go/env"
GOEXE=""
GOFLAGS=""
GOHOSTARCH="amd64"
GOHOSTOS="linux"
GONOPROXY=""
GONOSUMDB=""
GOOS="linux"
GOPATH="/root/go"
GOPRIVATE=""
GOPROXY="https://proxy.golang.org,direct"
GOROOT="/usr/local/go"
GOSUMDB="sum.golang.org"
GOTMPDIR=""
GOTOOLDIR="/usr/local/go/pkg/tool/linux_amd64"
GCCGO="gccgo"
AR="ar"
CC="gcc"
CXX="g++"
CGO_ENABLED="1"
GOMOD=""
CGO_CFLAGS="-g -O2"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-g -O2"
PKG_CONFIG="pkg-config"
GOGCCFLAGS="-fPIC -m64 -pthread -fmessage-length=0 -fdebug-prefix-map=/tmp/go-build000170528=/tmp/go-build -gno-record-gcc-switches"

**Additional context**
This really a serious issue, how if that happen to your 10mm? when you try to undelegate using CLI, it show messages;
have 0 want 1xxxx, it's can make heart attack, how to report if this happen on mainnet, lucky on my case we still used testnet token and also on dashboard it still show the messages

thankyou

**Comment by mindstyle85** [comment]
same as in your other issue.....

**Comment by sophoah** [comment]
@scraido on the passphrase thing at step whenever you are imported a new wallet via seed phrases, you have to set a passphrase yourselves, similar to doing it via the chrome extension. By default there aren't any

I haven't tested but similar to all command in hmy cli that need a passphrase if you do :
7.  Import an account using the mnemonic. Prompts the user to give the mnemonic.
hmy keys recover-from-mnemonic <ACCOUNT_NAME> --passphrase

you should be able to enter a passphrase to set it.

so basically on step 7 you imported your wallet without passphrase which explain why you don't need to type any in your test.

Now regarding your test, your step 3 most likely worked and your fund are in the undelegation phase waiting for 7 epoch which would explain why your fund was not in your wallet.

I don't see any issues here.
